src/app/app.component.html

```diff
@@ -0,0 +1,4 @@
+<app-header [name]="name"></app-header>
+<app-form (eventClicked)="headerChanged($event)"></app-form>
```

Comment:
where is the definition of  eventCLicked. use (click) instead

Reply:
I've created an Output() EventEmitter in form.component.ts. Whenever the heading is changed header an event is emitted from changeHeading() method in form.component.ts